src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/CommonModels/Enums.cs

```diff
@@ -316,4 +316,22 @@ public enum ILRAction
         Extend,
         Terminate,
     }
+
+    /// <summary>
+    /// Options to resolve conflict for a file share
+    /// </summary>
+    public enum ResolveConflict
```

Comment:
I think these options should be prefixed as well. Think "RestoreFSResolveConfictOption"